### ISSUE TITLE
Dropdown submenu not working

### ISSUE THREAD

**Issue by gluttm** [erdkse/adminlte-3-vue#13]
first, thank you for the work, menu submenu dropdown is not working, is there a soluction?

**Comment by mauriciopfl** [comment]
Hello, I had the same problem and solved it with the following steps:

1.  I downloaded the latest version of Admin-lte3 directly from the official website https://github.com/ColorlibHQ/AdminLTE/releases

2. I copied the files (dist/js/adminlte.js and dist/js/demo.js) to my vue project folder ./assets/js/
![image](https://user-images.githubusercontent.com/12702629/105110860-5f970f00-5a9e-11eb-853c-38863a0bed67.png)

3. install jquery on your vue project ( yarn add jquery or npm install jquery)


4. change the demo.js file and add the line:
   var jQuery = require ('jquery');
   like this:   
![image](https://user-images.githubusercontent.com/12702629/105110925-82c1be80-5a9e-11eb-8a28-ca6b1da04caa.png)
   
5. in the Main.vue file add inside the script:
     require ('../ assets / js / adminlte')
     require ('../ assets / js / demo')
     
![image](https://user-images.githubusercontent.com/12702629/105110965-98cf7f00-5a9e-11eb-9e16-5279b33c3f97.png)

     

It worked for me, I hope it works for you too
